Header header

float64 x
float64 y
float64 z
float64 x_rate
float64 y_rate
float64 z_rate
float64 x_acc
float64 y_acc
float64 z_acc